test/browser/features/back-button-post-session.feature

```diff
@@ -0,0 +1,25 @@
+@mock-errors @mock-api:address-success
+Feature: Error handling
+
+  API Errors in middle of journey
+
+  Background:
+    Given Back Button Betty is using the system
+    And they have started the address journey
+    And they searched for their postcode "E1 8QS"
+    Then they should see the results page
+    And they have selected an address ""
+    Then they should see the address page
+    Given they are on the address page
+    When they add their residency date "2000"
+    And they continue to confirm address
+    And they should see the confirm page
+    When they confirm their details
+    Then they should be redirected as a success
+
+
+  @mock-api:address-success
+  Scenario: Back button history
+    Given they have been redirected as a success
+    When they return to a previous page
+    Then they should be redirected as an error "general error"
```

Comment:
I was confused when I first saw this, as I thought it was redirecting as `error=general error`, not `error=server_error & error_description=general error`

Perhaps this could be reworded to make that clearer?